EIPS/eip-6110.md

```diff
@@ -60,58 +61,25 @@ The structure denoting the new deposit operation consists of the following field
 4. `signature: Bytes96`
 5. `index: uint64`
 
-RLP encoding of deposit structure **MUST** be computed in the following way:
+Deposits are a type of [EIP-7685](./eip-7685.md) request, therefore the encoding of the structure must be computed using the type byte `0x00`:
 
 ```python
-rlp_encoded_deposit = RLP([
-    RLP(pubkey),
-    RLP(withdrawal_credentials),
-    RLP(amount),
-    RLP(signature),
-    RLP(index)
+encoded_deposit = 0x00 ++ rlp([
```

Comment:
The problem with applying general request format to deposits is that CL and EL need to parse deposit data from their block structures, and each layer use own encoding for the same data (SSZ vs RLP).

Reply:
Maybe it's unclear, but this is only the encoding for the EL.

Reply:
Thanks for clarification! Now it makes more sense